contracts/invariant.ral

```diff
@@ -299,21 +299,22 @@ Contract Invariant(
         let (lowerChunk, upperChunk) = tickmap.getChunks(lowerTick, upperTick, tickSpacing)
 
         if (lowerChunk == upperChunk) {
-            tickmap.initializeChunk{caller -> ALPH: 1 alph}(caller, poolKey, lowerChunk)
+            tickmap.initializeChunk{caller -> ALPH: mapEntryDeposit!()}(caller, poolKey, lowerChunk)
         } else {
-            tickmap.initializeChunk{caller -> ALPH: 1 alph}(caller, poolKey, lowerChunk)
-            tickmap.initializeChunk{caller -> ALPH: 1 alph}(caller, poolKey, upperChunk)
+            tickmap.initializeChunk{caller -> ALPH: mapEntryDeposit!()}(caller, poolKey, lowerChunk)
+            tickmap.initializeChunk{caller -> ALPH: mapEntryDeposit!()}(caller, poolKey, upperChunk)
```

Comment:
I think it will be better to invoke `mapEntryDeposit!()` once and assign it to a variable than calling the macro few times in function body.